config/services.yaml

```diff
@@ -135,6 +135,11 @@ services:
             string $passwordExpiryWarn: "%env(ACCOUNT_PASSWORD_EXPIRATION_WARN)%"
             string $hostName: "%env(HOSTNAME)%"
 
+    App\Command\PelagosClearErrorAndFileCommand:
+        bind:
+            string $homedirPrefix: "%env(HOMEDIR_PREFIX)%"
+            string $dataStoreDirectory: "%env(DATA_STORE_DIRECTORY)%"
+
```

Comment:
These won't we needed if you inject Datastore, it can determine is a file is in the store.